apps/test-suite/TestUtils.js

```diff
@@ -68,17 +68,18 @@ export function getTestModules() {
     require('./tests/FBBannerAd'),
   ];
   if (ExponentTest && !ExponentTest.isInCI) {
+    const possiblyRunningInDeviceFarm = Constants.isDevice && Platform.OS === 'android';
```

Comment:
I'm not sure that you need this variable given that `ExponentTest.isInCI` tells us for certain whether the test is running in CI.

Reply:
If that were true then this PR wouldn't exist. These tests are running in device farm

Reply:
Make sure the `isInCI` variable is being set correctly.

See https://github.com/expo/expo/blob/master/fastlane/Fastfile#L224 and https://github.com/expo/expo/blob/master/android/expoview/src/main/java/versioned/host/exp/exponent/modules/test/ExponentTestNativeModule.java#L61

If that is working properly, the check for `Constants.isDevice && Platform.OS === 'android'` should be redundant.